src/main/java/io/github/jelastic/core/config/AuthConfiguration.java

```diff
@@ -33,4 +33,10 @@
 
     @NotNull @NotEmpty
     private String password;
+
+    private boolean tlsEnabled;
+
+    private String trustStorePath;
```

Comment:
Do you want to leave a validation here if tlsConfig is set to true then the trustStore paths shouldn't be null or empty?